app/views/sleeplogs/_form.html.haml

```diff
@@ -1,14 +1,19 @@
 #sleeplog
   = simple_form_for @sleeplog, remote: true do |f|
     .row
-      %p= f.input :date, as: :date,
-                    start_year: Time.now.year - 1,
-                    end_year: Time.now.year,
-                    start_month: Time.now.month - 1,
-                    end_month: Time.now.month,
-                    order: [:month, :day, :year]
-    .row
-      %p= f.input :hours, as: :range, min: "0", max: "24"
+      .small-2.columns
+        = f.date_field :date
+    .row
+      .small-8.small-centered.columns
+        .range-slider{"data-options" => "start: 0; end: 16; display_selector: #sliderOutput", "data-slider" => ""}
+          %span.range-slider-handle{:role => "slider", :tabindex => "0"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace